routes/auth.js

```diff
@@ -0,0 +1,42 @@
+const { User } = require("../models/user");
+const _ = require("lodash");
+const bcrypt = require("bcrypt");
+const Joi = require("joi");
+const express = require("express");
+const router = express.Router();
+
+router.post("/", async (req, res) => {
+  const { error } = validate(req.body);
+  if (error) return res.status(400).send(error.details[0].message);
```

Comment:
favor multilines over single lines and always use the curly braces (`{}`)

```js
if (error) {
  // your code
}
```

apply this rule to everything